### PR TITLE
Knative eventing mesh application broker image bump

### PR DESCRIPTION
**Pull request by nachtmaar**
`pre-master-kyma-gke-upgrade` job is failing because of this open issue: https://github.com/kyma-project/kyma/issues/6686

**Description**

Changes proposed in this pull request:

- Bump image for code changes in https://github.com/kyma-project/kyma/pull/6636
- Run tests for code changes in https://github.com/kyma-project/kyma/pull/6636

**Related issue(s)**

#6148

### CODE REVIEW

File: resources/application-connector/charts/application-broker/templates/cluster-role.yaml [hunk 1]
```diff
@@ -29,4 +30,11 @@ rules:
 - apiGroups: ["servicecatalog.k8s.io"]
   resources: ["servicebrokers"]
   verbs: ["get", "create", "delete", "list", "update"]
-
+# subscriptions are created by the application-broker to enable events flowing from a channel to the broker
+- apiGroups: ["messaging.knative.dev"]
+  resources: ["subscriptions"]
+  verbs: ["get", "create", "delete", "list", "update"]
```
Comment (antoineco) [inline resources/application-connector/charts/application-broker/templates/cluster-role.yaml]: `get` is not used in the broker if I'm not mistaken.

File: resources/application-connector/charts/application-broker/templates/cluster-role.yaml [hunk 2]
```diff
@@ -29,4 +30,11 @@ rules:
...
+# readonly access to channels is required to create a subscription (channel is part of subscription spec)
+- apiGroups: ["messaging.knative.dev"]
+  resources: ["channels"]
+  verbs: ["get", "list"]
```
Comment (antoineco) [inline resources/application-connector/charts/application-broker/templates/cluster-role.yaml]: Same here, `get` doesn't seem to be used anywhere in the broker.

File: resources/application-connector/charts/application-broker/templates/cluster-role.yaml [hunk 3]
```diff
@@ -19,7 +19,8 @@ rules:
   verbs: ["get", "create", "delete", "update"]
 - apiGroups: [""]
   resources: ["namespaces"]
-  verbs: ["patch", "list", "watch"]
+  # write access is required because namespaces need to be annotated in order to create a knative broker
+  verbs: ["patch", "list", "watch", "get", "update"]
```
Comment (antoineco) [inline resources/application-connector/charts/application-broker/templates/cluster-role.yaml]: I know it was already there before your change, but we _may_ be able to remove `patch`.

Comment (nachtmaar) [reply resources/application-connector/charts/application-broker/templates/cluster-role.yaml]: Sure, no problem. Works without watch. Minimum permissions are always good 👍

Comment (antoineco) [reply resources/application-connector/charts/application-broker/templates/cluster-role.yaml]: watch or Patch?

Comment (nachtmaar) [reply resources/application-connector/charts/application-broker/templates/cluster-role.yaml]: Fixed

File: resources/application-connector/charts/application-broker/templates/cluster-role.yaml (hunk 2 above)
Comment (nachtmaar) [reply resources/application-connector/charts/application-broker/templates/cluster-role.yaml]: Fixed

File: resources/application-connector/charts/application-broker/templates/cluster-role.yaml (hunk 1 above)
Comment (nachtmaar) [reply resources/application-connector/charts/application-broker/templates/cluster-role.yaml]: Fixed

**Comment by nachtmaar** [comment]
@antoineco

**Comment by Abd4llA** [comment]
I'm passing this on a temporary stage to unblock the release given that tests are succeeding and it received an initial round of review. a later deeper post-merge code review will take place

**Pull request merged**